docs/website/blog/2023-06-14-dlthub-tongtest.md

```diff
@@ -0,0 +1,166 @@
+# Replace your_username with actual username
+db = DeepLake(dataset_path="hub://"your_username"/dlt_gpt", read_only=True, embedding_function=embeddings)
+
+# Create a retriever
+retriever = db.as_retriever()
+retriever.search_kwargs['distance_metric'] = 'cos'
+retriever.search_kwargs['fetch_k'] = 100
+retriever.search_kwargs['maximal_marginal_relevance'] = True
+retriever.search_kwargs['k'] = 10
+```
+#### 7. Initialize the GPT model 
+```python 
+from langchain.chat_models import ChatOpenAI
+from langchain.chains import ConversationalRetrievalChain
+
+model = ChatOpenAI(model_name='gpt-3.5-turbo') 
+qa = ConversationalRetrievalChain.from_llm(model,retriever=retriever)
+```
+### III.                                                             Result   
+After the walkthrough, we can start to experiment different questions and it will output answers based on our training from dlt hub repository. 
+ 
+Here, I asked ‘’why should data teams use dlt?’’ 
+
+![chatgptq1](\img\chatgptQ1.png)  
+
+It outputted:
+
+1. It works seamlessly with Airflow and other workflow managers, making it easy to modify and maintain your code.
+2. You have complete control over your data. You can rename, filter, and modify it however you want before it reaches its destination.
+
+
+Next, I asked "Who is dlt for?"  
+
+![chatgptq2](\img\chatgptQ2..png)  
+
+It outputted:
+1. `dlt` is meant to be accessible to every person on the data team, including data engineers, analysts, data scientists, and other stakeholders involved in data loading. It is designed to reduce knowledge requirements and enable collaborative working between engineers and analysts.
+
+### IV. Summary  
+ 
+In this article, "Training GPT with an Open-Source `dlt` Repository," we can see how the `dlt` library can easily retrieve data from the `dlt` hub repository. Then, by simply training the model, we can get personalized results.
+
+It can also be used in different places. For more understanding about the `dlt` library or how to use it in different places, check these [user guide pages](https://dlthub.com/docs/user-guides) to discover more about the potential of `dlt` and how it can revolutionize your workflow. Also, take the plunge and join our vibrant [Slack community](https://dlthub-community.slack.com/ssb/redirect) to delve deeper into the `dlt` and engage in stimulating discussions!
+
+
+***Curious? Give the Colab demo a try or share your questions with us, and we'll have ChatGPT address them in our upcoming article.***
+
+*** 
+> [ About dlt ]
+
+`dlt` is an open-source library that enables you to create a data [pipeline](https://dlthub.com/docs/general-usage/glossary#pipeline) in a Python script. To use it, `pip install dlt` and then `import dlt`. Once set up, it will automatically load any [source](https://dlthub.com/docs/general-usage/glossary#source) (e.g., an API) into a live dataset stored in the [destination](https://dlthub.com/docs/general-usage/glossary#destination) of your choice.
```

Comment:
Replace this with call to action
you can keep a brief info about dlt, but also say
- learn more about dlt here - link to intro
- need help or want to discuss? join our slack community LINK